libs/labelbox/src/labelbox/schema/project.py

```diff
@@ -1746,6 +1747,63 @@ def __check_data_rows_have_been_processed(
         return response["queryAllDataRowsHaveBeenProcessed"][
             "allDataRowsHaveBeenProcessed"]
 
+    def get_overview(self) -> Project_Overview:
```

Comment:
Should there be a unit test for this?

Reply:
The current test (`test_get_overview()`) is part of `libs/labelbox/tests/integration/test_task_queue.py` because it is related since the new method provides the number of data rows per task queue.